app/repositories/application-repository.js

```diff
@@ -19,15 +19,26 @@ async function getByEmail (email) {
     })
 }
 
-async function getAll (page = 0) {
-  return models.application.findAll(
-    {
-      order: [['createdAt', 'DESC']],
-      limit: 20,
-      offset: page
-    })
+async function getAll (limit, offset, sbi) {
```

Comment:
Consider providing some defaults for the limit and offset rather than requiring the callee to do that.

Also consider if adding a JSDoc would be useful please.